src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -724,8 +724,9 @@ private static final class CliOptions {
         /** Option that controls whether to print the AST of the file. */
         @Option(names = {"-t", "--tree"},
                 description = "Prints Abstract Syntax Tree(AST) of the checked file. The option "
-                        + "cannot be used other options and requires exactly one file to run on "
-                        + "to be specified.")
+                        + "cannot be used with other options and requires exactly one file to "
+                        + "run on to be specified and this file must be located in the same "
+                        + "directory as the checkstyle JAR.")
```

Comment:
this is not true:
`/var/tmp$ java -jar checkstyle-10.12.4-all.jar -T ~/java/github/checkstyle/checkstyle/src/main/java/com/puppycrawl/tools/checkstyle/DetailAstImpl.java`

Reply:
Sorry, There was an extension to these lines in another commit. I have no idea why it is missing. It must've got dropped during rebase. Let me fix this

Reply:
>and this file must be located in the same directory as the checkstyle

This should be removed